charts/argo-cd/values.yaml

```diff
@@ -2588,7 +2588,6 @@ server:
     termination_policy: None
 
   # Gateway API HTTPRoute configuration
-  # NOTE: Gateway API support is in EXPERIMENTAL status
```

Comment:
Lets keep this for now. (Still more or less experimental since there is no upstream support by the project itself - lacks docs).